package.json

```
@@ -19,7 +20,10 @@
 		"lint": "tsc && xo && stylelint \"css/**/*.css\"",
 		"test": "npm run lint",
 		"start": "tsc && electron .",
-		"build": "tsc",
+		"build": "tsc && electron-builder --dir",
```

This would break our GitHub pipelines ([action-electron-builder](https://github.com/marketplace/actions/electron-builder-action)). The build script should only contain commands for compiling TS code, not for building the app.
```suggestion
		"build": "tsc",
```

---

Ahh ok. Will remove